_data/social.yml

```diff
@@ -261,7 +261,7 @@ websites:
       doc: https://vk.com/page-59800369_47885415
 
     - name: Weibo
-      url: http://www.weibo.com
+      url: https://www.weibo.com/
```

Comment:
I'm unsure if they provide HTTPS on their regular site or only the overseas version. 🤔

Reply:
Checked on VPN from Hong Kong and they provided the site over HTTPS however the mainland could have a different setup. Also, it's impossible to change language as that option is in the footer and their JS-script autoloads more content when the screen reaches the footer 😠

Reply:
https:// is the direct responseUri I get from the HTTP-request, the forward to /overseas is in a later step, so there must be some form of https involved.